test/models/user_test.rb

```diff
@@ -298,14 +298,14 @@ def setup
     user = build(:user, role: 'organisation_admin', organisation_id: nil)
 
     refute user.valid?
-    assert_equal "can't be 'None' for an Organisation admin", user.errors[:organisation_id].first
+    assert_equal "can't be 'None' for Organisation Admin", user.errors[:organisation_id].first
   end
 
   test "super organisation admin must belong to an organisation" do
     user = build(:user, role: 'super_organisation_admin', organisation_id: nil)
 
     refute user.valid?
-    assert_equal "can't be 'None' for a Super Organisation admin", user.errors[:organisation_id].first
+    assert_equal "can't be 'None' for Super Organisation Admin", user.errors[:organisation_id].first
```

Comment:
This change belongs in a previous commit as you didn't change the error message in this commit.